### PR TITLE
Add i18n translation for global usages

### PR DESCRIPTION
**Pull request by WindSekirun**
This PR make 'evobot' send message in various language.

**Step to add new language**
1. Copy `en.json` to other file, with renaming '{language}.json'. (if you want to translate Korean, it will be ko.json)
2. Translate renamed file
3. In index.js (root directory), add target language into 'locales' array (just like 12d9a73ead6b861db175c32c07125346b5ecb03b)

If json isn't valid (following [RFC8259](https://tools.ietf.org/html/rfc8259). you can check validating by jsonformatter or other sites), it won't compile or running.

**OBJECTIVE**
* add code for [i18n-node](https://github.com/mashpie/i18n-node)
* add base translation file (en.json)
* add Korean (ko) translation file (ko.json)
* add French (fr) translation file (fr.json) credit to @totolerigolo324

**TODOS**
* [x] change hard-coded strings into base translation file
* [x] Translating korean
* [x] check working: help
* [x] check working: loop
* [x] check working: lyrics
* [x] check working: nowplaying
* [x] check working: pause
* [x] check working: play
* [x] check working: playlist
* [x] check working: pruning
* [x] check working: queue
* [x] check working: remove
* [x] check working: resume
* [x] check working: search
* [x] check working: shuffle
* [x] check working: skip
* [x] check working: skipto
* [x] check working: stop
* [x] check working: uptime
* [x] check working: volume
* [x] check working: clip
* [x] check working: clips
* [x] check working: invite
* [x] check working: ping
* [x] check working: move

### CODE REVIEW

**Comment by wolf-yuan-6115** [comment]
Hi !
Your pull request is very good!!!
Thank u!

**Comment by WindSekirun** [comment]
I finished my work of adding translation. Can you review this? @eritislami

**Comment by eritislami** [comment]
Resolved merge conflicts, if you don't mind can you also add a translation for this line

https://github.com/eritislami/evobot/blob/6546b3750e999183b2b88c985afcb4017f2a2529/include/play.js#L17

File: include/play.js [hunk 1]
```diff
@@ -1,187 +1,201 @@
...
+    const queue = message.client.queue.get(message.guild.id);
+
+    if (!song) {
+      setTimeout(function () {
+        if (queue.connection.dispatcher && message.guild.me.voice.channel) return;
+        queue.channel.leave();
+        queue.textChannel.send(i18n.__("play.leaveChannel"));
+      }, STAY_TIME * 1000);
+      return queue.textChannel.send(i18n.__("play.queueEnded")).catch(console.error);
+      return message.client.queue.delete(message.guild.id);
```
Comment (WindSekirun) [inline include/play.js:20]: @eritislami Sure! i added translation for 'Leaving the channel...' which related to #351.

But, It seems 'message.client.queue.delete(message.guild.id)' is unreachable due to `return` of L19, 

So, how about modifying it as below?

```
if (!song) {
      setTimeout(function () {
        if (queue.connection.dispatcher && message.guild.me.voice.channel) return;
        queue.channel.leave();
        queue.textChannel.send(i18n.__("play.leaveChannel"));
      }, STAY_TIME * 1000);
      message.client.queue.delete(message.guild.id);
      return queue.textChannel.send(i18n.__("play.queueEnded")).catch(console.error);
    }
```

**Comment by WindSekirun** [comment]
@eritislami What else do i need to do before merge?

**Comment by eritislami** [comment]
> @eritislami What else do i need to do before merge?

Nothing it's all good i just need some time

**Comment by WindSekirun** [comment]
@eritislami I added translation for 'clip', 'clips', 'invite', 'ping' which recently implemented. (and feature is working too.) Can you check? or I need another waiting time?

**Review by totolerigolo324 — CHANGES_REQUESTED** [review]
I did the translation in French. Interested in me adding it? (I have already made the fr.json file)

**Comment by WindSekirun** [comment]
@totolerigolo324 I also add translation for 'move' command.

```
"move": {
    "description": "Move songs around in the queue",
    "errorNotQueue": "There is no queue.",
    "usagesReply": "Usage: {prefix}move <Queue Number>",
    "result": "<@{author}> 🚚 moved **{title}** to {index} in the queue."
  },
```

When attaching fr.json, it would be nice if `move` translation was also added.

**Comment by totolerigolo324** [comment]
Here is the fr.json: https://gist.github.com/totolerigolo324/235a93b35cadd4806e9125b6f89e0624

**Comment by WindSekirun** [comment]
@totolerigolo324 Done, by 12d9a73ead6b861db175c32c07125346b5ecb03b.

**Review by wolf-yuan-6115 — APPROVED** [review]

**Comment by eritislami** [comment]
@WindSekirun & @totolerigolo324 
Thank you for the work you did on this, i finally had some time to merge it all together.

**Pull request merged**